pkg/apis/extensions/v1beta1/types.go

```diff
@@ -933,6 +933,15 @@ type PodSecurityPolicySpec struct {
 	// host paths may be used.
 	// +optional
 	AllowedHostPaths []string `json:"allowedHostPaths,omitempty" protobuf:"bytes,15,opt,name=allowedHostPaths"`
+	// DefaultAllowPrivilegeEscalation controls the default setting for whether a
+	// process can gain more privileges than it's parent process.
+	// // +optional
```

Comment:
I hope this is the source location, because those are propagated to all auto-generated snippets:
 * s/it's parent/its parent/
 * Spurious double slash before the optional annotation

The first typo is also in PR comment / release note, but I can't comment directly there :)

Reply:
@jessfraz I see that s/it's parent/its parent/ wasn't fixed. Do you want me to prepare PR for that?

Reply:
I've submitted https://github.com/kubernetes/kubernetes/pull/51536